app/views/static_pages/cookies.html.erb

```diff
@@ -130,7 +130,7 @@
         <tr class="govuk-table__row">
           <td class="govuk-table__cell">_dfe_teachers_payment_service_session</td>
```

Comment:
hmm - this seems to be the wrong cookie name - should be `_claim_session` - is there another ticket to address this?

Reply:
if not, defo raise one

Reply:
raised: https://dfedigital.atlassian.net/browse/CAPT-1774